package.json

```diff
@@ -29,16 +29,18 @@
   "license": "MIT",
   "homepage": "https://github.com/companieshouse/overseas-entities-web#readme",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.85",
+    "@companieshouse/api-sdk-node": "github:companieshouse/api-sdk-node#roe-2275-remove-entity-poc",
     "@companieshouse/node-session-handler": "^4.1.9",
     "@companieshouse/structured-logging-node": "^1.0.8",
     "@companieshouse/web-security-node": "^2.0.3",
     "cookie-parser": "^1.4.6",
     "express": "^4.17.3",
     "express-validator": "^6.14.0",
     "govuk-frontend": "^4.0.1",
+    "install": "^0.13.0",
```

Comment:
i think the "npm" and "install" packages may have been added by accident, possibly by running 'npm install npm install' on the command line - so can probably be ignored if we use this POC in future